test/unit_tests/apps/ccdb-landing-map/js/Chart-spec.js

```diff
@@ -1,5 +1,7 @@
 import 'regenerator-runtime/runtime';
-import Chart from '../../../../../cfgov/unprocessed/apps/ccdb-landing-map/js/Chart';
+import
+Chart
+  from '../../../../../cfgov/unprocessed/apps/ccdb-landing-map/js/Chart';
```

Comment:
```suggestion
import {
  Chart
} from '../../../../../cfgov/unprocessed/apps/ccdb-landing-map/js/Chart';
```

Reply:
I am updating all import lines to use curly brace formatting, even if there is only one item, for two reasons:
1. Consistency in that all imports have the same formatting
2. Import lines should not be difficult to read